src/App.jsx

```diff
@@ -0,0 +1,62 @@
+import { useEffect, useState } from 'react';
+
+import useInput from './hooks/useInput';
+
+import nothingContents from './utils/nothingContents';
+
+import Todo from './pages/Todo';
+
+export default function App() {
+  const [todo, todoHandler, setTodo] = useInput('');
+  const [disabled, setDisabled] = useState(false);
+
+  const { inputValue: inputTodo, contents: todos } = todo;
+
+  useEffect(() => {
+    if (inputTodo === '') {
+      setDisabled(true);
+    }
+    if (inputTodo !== '') {
+      setDisabled(false);
+    }
+  }, [inputTodo, setDisabled]);
```

Comment:
```suggestion
  useEffect(() => {
      setDisabled(inputTodo === '')
  }, [inputTodo]);
```

이렇게도 표현이 가능할 것 같아요.

참고로 의존성 배열에 setState 함수를 넣어주지 않아도 돼요. 공식문서를 참고해보세요!

https://reactjs.org/docs/hooks-reference.html#basic-hooks

Reply:
아하 감사합니다!! ㅎㅎㅎ 이런 방법이 있는데..!
조금만 더 생각하고 코드를 작성했으면 놓치지 않았을 부분인데
놓치고 말았네요..!
ㅎㅎㅎ 피드백 감사합니다!